lib/features/children/add_member/cubit/add_member_state.dart

```diff
@@ -0,0 +1,40 @@
+part of 'add_member_cubit.dart';
+
+sealed class AddMemberState extends Equatable {
+  const AddMemberState({
+    this.child,
+  });
+
+  final Child? child;
+
+  @override
+  List<Object?> get props => [child];
+}
```

Comment:
Wouldn't be easier to make a single state for this cubit? instead of just passing around the "child"?

Reply:
oh yeah, with the status! Thanks!